environments/production/postgresql.yml

```diff
@@ -23,6 +23,12 @@ pg_repack:
       password: "{{ postgres_users.root.password }}"
       port: 5432
       skip_superuser_check: True
+    - database: formplayer
+      host: pgformplayer0-production.cl9dmuo3ok4h.us-east-1.rds.amazonaws.com
```

Comment:
Reason for using host: pgformplayer0-production.cl9dmuo3ok4h.us-east-1.rds.amazonaws.com is that the playbook is unable to identify the host rds_pgformplayer0 therefore had to use the DNS name

```
TASK [Create pg_repack extension] *******************************************************************************************
 ok: [10.202.40.153] => (item={'database': 'commcarehq_synclogs', 'host': 'pgsynclog0-production.cl9dmuo3ok4h.us-east-1.rds.amazonaws.com', 
 'password': '*************************', 'port': 5432, 'skip_superuser_check': True, 'username': 'root'}) 
failed: [10.202.40.153] (item={'database': 'formplayer', 'host': 'rds_pgformplayer0', 'password': '*************************', 'port': 5432, 'skip_superuser_check': True, 'username': 'root'}) 
=> {"ansible_loop_var": "item", "changed": false, 
"item": {"database": "formplayer", "host": "rds_pgformplayer0", 
"password": "*************************", "port": 5432, "skip_superuser_check": true, "username": "root"}, 
"msg": "unable to connect to database: could not translate host name \"rds_pgformplayer0\" to address: Name or service not known\n"}

````